pointerevents/pointerevent_on_event_handlers.html

```diff
@@ -0,0 +1,72 @@
+    <h4>Test Description: This test checks if on pointer event handlers through attributes works properly using synthetic pointerevents. For valid results, this test must be run without generating real (trusted) events on the black rectangle below.</h4>
+    <div id="target0"
+         onpointercancel="window.eventHappened = 'pointercancel';"
+         onpointerdown="window.eventHappened = 'pointerdown';"
+         onpointerup="window.eventHappened = 'pointerup';"
+         onpointermove="window.eventHappened = 'pointermove';"
+         onpointerout="window.eventHappened = 'pointerout';"
+         onpointerover="window.eventHappened = 'pointerover';"
+         onpointerleave="window.eventHappened = 'pointerleave';"
+         onpointerenter="window.eventHappened = 'pointerenter';"
+         ongotpointercapture="window.eventHappened = 'gotpointercapture';"
+         onlostpointercapture="window.eventHappened = 'lostpointercapture';"
+    ></div>
+    <script>
+        window.eventHappened = '';
+
+        All_Pointer_Events.forEach(function(event) {
+            var on_event = "on" + event;
+            console.log(on_event);
+
+            test(function() {
+                for (const location of [window, HTMLElement.prototype, SVGElement.prototype, Document.prototype]) {
+                  assert_true(location.hasOwnProperty(on_event),
+                    `${location.constructor.name} has an own property named `+ on_event);
+                }
+            }, on_event + " is on the appropriate locations for GlobalEventHandlers");
+            test(function {
+              const htmlElement = document.createElement("span");
+              const svgElement = document.createElementNS("http://www.w3.org/2000/svg", "g");
+              for (const location of [window, htmlElement, svgElement, document]) {
+                assert_equals(location.onauxclick, null,
+                  `The default value of the property is null for a ${location.constructor.name} instance`);
+              }
+            }, "The default value of " + on_event + " is always null");
+
+            test(function {
+              window.eventHappened = '';
+              const element = document.querySelector("#target0");
+              const compiledHandler = element[on_event];
+              assert_equals(typeof compiledHandler, "function", "The " + on_event + " property must be a function");
+              compiledHandler();
+              assert_equals(window.eventHappened, event, "Calling the handler must run the code");
+            }, "The " + on_event + " content attribute must be compiled into the " + on_event + " property");
+
+            test(function {
+              const element = document.createElement("meta");
+              element[on_event] = function(e) {
+                assert_equals(e.currentTarget, element, "The event must be fired at the <meta> element");
+              };
+              element.dispatchEvent(new Event(event));
```

Comment:
Yeah. Makes sense. I changed it to an async test so if the handler doesn't fire that test times out.